style.css

```diff
@@ -283,7 +283,7 @@ body {
   border-top-right-radius: 98px;
   margin-top: 198px;
   height: 1132px;
-  width: 100%;
+  width: 99%;
```

Comment:
- [ ] as the previous CR mentioned there is still a horizontal overflow between screen widths above 400px and below 770px, this causes some of the text to be invisible. please fix this.
![afbeelding](https://github.com/josman227/portfolio.github.io/assets/39240820/e8e13662-76a0-4031-9ba8-bb9e74c7a91e)